x/feerefunder/keeper/keeper.go

```diff
@@ -60,14 +60,20 @@ func (k Keeper) LockFees(ctx context.Context, payer sdk.AccAddress, packetID typ
 
 	k.Logger(c).Debug("Trying to lock fees", "packetID", packetID, "fee", fee)
 
-	if _, ok := k.channelKeeper.GetChannel(c, packetID.PortId, packetID.ChannelId); !ok {
-		return errors.Wrapf(channeltypes.ErrChannelNotFound, "channel with id %s and port %s not found", packetID.ChannelId, packetID.PortId)
+	params := k.GetParams(c)
+	if !params.FeeEnabled {
+		k.Logger(c).Debug("Skip locking fees because fee disabled", "packetID", packetID, "fee", fee)
+		return nil
 	}
 
 	if err := k.checkFees(c, fee); err != nil {
 		return errors.Wrapf(err, "failed to lock fees")
```

Comment:
not your code but still would be good to rephrase
```suggestion
		return errors.Wrapf(err, "fees check failed")
```